### ISSUE TITLE
CLI still fails

### ISSUE THREAD

**Issue by bertsky** [qurator-spk/eynollah#13]
Another one surfaced:

```
2 [[3.6625916e-17 1.0000000e+00 8.9200320e-18 2.0860562e-30 3.3768879e-20
  2.0703751e-30]] num_col_classifier
300
False is_image_enhanced
enhancing: 12.93634581565857
97.62038955952693 ratio_of_two_models
textregion: 238.8552384376526
textline: 301.728711605072
-0.8484848484848477 slope_deskew
deskewing: 311.6365988254547
marginals: 312.0726170539856
2 num_col_classifier
boxes in: 313.63718938827515
Traceback (most recent call last):
  File "/data/ocr-d/eynollah/sbb_newspapers_org_image/eynollah.py", line 3635, in run
    self.write_into_page_xml(txt_con_org, page_coord, self.dir_out, order_text_new, id_of_texts_tot, all_found_texline_polygons, all_box_coord, polygons_of_images, polygons_of_marginals, all_found_texline_polygons_marginals, all_box_coord_marginals, self.curved_line, slopes, slopes_marginals)
  File "/data/ocr-d/eynollah/sbb_newspapers_org_image/eynollah.py", line 1941, in write_into_page_xml
    page_print_sub = ET.SubElement(page, "PrintSpace")
TypeError: SubElement() argument 1 must be xml.etree.ElementTree.Element, not lxml.etree._Element
```

**Comment by bertsky** [comment]
Seems like you need to put back the code in the comments here:
https://github.com/qurator-spk/eynollah/blob/85d631aadf47b7f6c6a88288b73b931efcb561a4/sbb_newspapers_org_image/eynollah.py#L32

**Comment by kba** [comment]
> Seems like you need to put back the code in the comments here:
> https://github.com/qurator-spk/eynollah/blob/85d631aadf47b7f6c6a88288b73b931efcb561a4/sbb_newspapers_org_image/eynollah.py#L32

I appreciate you trying this out but please give me a few more days, the state in the branch I'm working on has changed significantly from current main.

**Comment by bertsky** [comment]
> I appreciate you trying this out but please give me a few more days, the state in the branch I'm working on has changed significantly from current main.

Understood. If you are referring to https://github.com/qurator-spk/eynollah/commits/refactor-cntd – boy I don't envy you – then this might still be relevant though. (Also, if I could make a wish: replace `PrintSpace` with `Border`.)

Just FYI: The above change plus the 2 fixes mentioned in #11 suffice to actually get the current master working again.

**Comment by cneud** [comment]
Yeah we were too optimistic when we removed the lovely "warning: work-in-progress" gif before Christmas only to break master immediately after...much gratitude for @kba's refactoring though. 

`Border` will be `PrintSpace` same as in [sbb_textline_detection](https://github.com/qurator-spk/sbb_textline_detection/pull/38). I will add an issue just to make sure though. Much other renaming to do as well ;-)

Glad you got it working!
